paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormViewModel.kt

```diff
@@ -393,17 +335,6 @@ internal class USBankAccountFormViewModel @Inject internal constructor(
     }
 
     fun onDestroy() {
-        // Save before we die
-        _currentScreenState.update {
```

Comment:
How do you know this is no longer necessary?

Reply:
We don’t use the screen state properties that are being updated here anywhere. Any input is preserved in the controllers (for better or for worse).